linera-views/src/key_value_store_view.rs

```diff
@@ -146,20 +170,38 @@ where
         self.was_cleared = true;
         self.updates.clear();
         self.deleted_prefixes.clear();
+        self.total_size = (0, 0);
+        self.sizes.clear();
         *self.hash.get_mut() = None;
     }
 }
 
 impl<'a, C> KeyValueStoreView<C>
 where
-    C: Send + Context,
+    C: Send + Context + Sync,
     ViewError: From<C::Error>,
 {
     fn max_key_size(&self) -> usize {
         let prefix_len = self.context.base_key().len();
         C::MAX_KEY_SIZE - 1 - prefix_len
     }
 
+    /// Getting the total size that will be used when stored
```

Comment:
```suggestion
    /// Getting the total sizes that will be used for keys and values when stored
```

Maybe the `total_size` field (and other fields) should also have a comment to clarify it; makes it easier for a reader unfamiliar with the code.

Reply:
We typically do not comment field entries in `linera-views`. So, maybe, that should be part of a bigger effort.